functions/index.js

```diff
@@ -19,7 +19,9 @@ exports.createNewTask = functions.https.onRequest((request, response) => {
         var estimatedTime = request.body.data.EstimatedTime;
         var status = request.body.data.Status;
         var category = request.body.data.Category;
-        var taskIdNumber = getIdNumber();
+        var sprintId = request.body.data.SprintId;
+        var getSprintId = "S" + sprintId;
```

Comment:
Please change the name of the variable getSprintId to fullSprintId
A condition statement must be applied to check sprintId.

if(sprintId == -1)
fullSprintId = "Backlog"
else
fullSprintId = "S"+sprintId

Something like this.